resources/event-bus/templates/tests/test.yaml

```diff
@@ -27,6 +27,7 @@ spec:
             - --publish-event-uri=http://event-publish-service:8080/v1/events
             - --publish-status-uri=http://event-publish-service:8080/v1/status/ready
             - --subscriber-image={{ .Values.global.containerRegistry.path }}/{{ .Values.global.event_bus_tests.dir }}event-bus-e2e-subscriber:{{ .Values.global.event_bus_tests.version }}
```

Comment:
I know this has been there for quiet some time, but shouldn't we reference the image name "event-bus-e2e-subscriber" in values.yaml ? something like this https://github.com/kyma-project/kyma/blob/0e88f482af87dd1e89ad449fec2b6fdfe3eca748/resources/event-bus/charts/event-publish-service/templates/deployment.yaml#L33

Reply:
IMO, I don't see the benefit. In order to retain consistency with the other charts, I would change https://github.com/kyma-project/kyma/blob/0e88f482af87dd1e89ad449fec2b6fdfe3eca748/resources/event-bus/charts/event-publish-service/templates/deployment.yaml#L33